lib/core/src/lib/ui-header/ui-header.component.spec.ts

```diff
@@ -0,0 +1,112 @@
+/*!
+ * @license
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CommonModule } from '@angular/common';
+import { TranslateModule } from '@ngx-translate/core';
+import { UiHeaderComponent } from './ui-header.component';
+
+describe('UiHeaderComponent', () => {
+    let component: UiHeaderComponent;
+    let fixture: ComponentFixture<UiHeaderComponent>;
+
+    const getLogoImgElement = () => fixture.nativeElement.querySelector('.adf-toolbar-logo');
+    const getTitleElement = () => fixture.nativeElement.querySelector('.adf-toolbar-title');
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [CommonModule, TranslateModule.forRoot(), UiHeaderComponent]
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(UiHeaderComponent);
```

Comment:
I guess maybe we should move this to a single beforeEach(). Calling TestBed.configureTestingModule() after TestBed.createComponent() will [throw an error](https://v17.angular.io/guide/testing-components-basics#createcomponent). Having it in 2 seperate beforeEach blocks creates a possibility of that happening

Reply:
good catch, merged into one, ty